### DISCUSSION TITLE
Other serverless provider

### DISCUSSION THREAD

**Discussion by Foixa** [davefojtik/RunPod-Fooocus-API#28] [Ideas]
It would be nice to be able to use fooocus-api with another serverless provider.

Or if it had AWS Deeplearning Container support. https://github.com/aws/deep-learning-containers/blob/master/available_images.md

Maybe we should open a new branch for this and start.

**Comment by davefojtik** [comment]
The repo would probably have to be renamed to something like `Fooocus-API-Implementations` then.
But if it was up to me, I would merge everything together. We have [Fooocus](https://github.com/lllyasviel/Fooocus) and their Google Colab support, [Fooocus-API](https://github.com/mrhan1993/Fooocus-API) and Replicate support, [RunPod-Fooocus-API](https://github.com/davefojtik/RunPod-Fooocus-API) and other implementation repos. It's quite chaotic.

Personally, I think the best scenario would be Fooocus supporting the API natively, having a separate dev branch for it. And possibly other branches for individual implementations all under the original project, or an organization. I haven't discussed this with the maintainers yet, as it would mean **_a lot_** more management and cooperation for them. Also, each of us has a bit different standards for testing, updates and even the time we can give to these projects.

Currently, I don't have the interest or time to start another implementation. This is just a tool for our much more complex AI Agent and normally I have to work on proprietary software. If you don't feel like starting your own repo, I can give you a branch here and/or cooperation. But unfortunately, I can't promise active development on it.
